lib/fs.js

```diff
@@ -15,6 +15,7 @@ const internals = {
     }
 };
 
+const notFound = ['ENOENT', 'ENOTDIR'];
```

Comment:
Hapi style guide does not allow defining globals. You need to add it as a property on the internals object.

Additionally this needs to be a `Set`.

Reply:
ok, moved it to internals as Set